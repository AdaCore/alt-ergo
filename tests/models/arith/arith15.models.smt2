(set-option :produce-models true)
(set-logic ALL)
(declare-const x Int)
(declare-const y Int)
(assert (<= x 5))
(assert (<= 0 y))
(maximize (- (* 5 x) y))
(check-sat)
(get-model)
(get-objectives)
