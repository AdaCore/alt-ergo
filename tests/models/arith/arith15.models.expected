
unknown
(
  (define-fun x () Int 5)
  (define-fun y () Int 0)
)
(objectives 
  ((- (* 5 x) y) 25)
)